styles/variables.scss

```diff
@@ -46,3 +46,4 @@ $font-secondary-color: #777;
 
 $navbar-breakpoint: 1160px;
 $tablet-breakpoint: 768px;
+$single-column-breakpoint: 500px;
```

Comment:
What's the point of this variable? If we only use it in a couple of places, we can always just define it scoped in that file (instead of globally).

Reply:
hm okay I'll move it to the specific file then

Reply:
it's in here now since I thought it would come in useful later, but I guess we could always move it back to variables if we needed to